### ISSUE TITLE
New submissions for Thu, 16 Feb 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#39]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
## Keyword: medical imagingremote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
### Vulnerability Analysis of Digital Banks' Mobile Applications
 - **Authors:** Polra Victor Falade, Grace Bunmi Ogundele
 - **Subjects:** Cryptography and Security (cs.CR)
 - **Arxiv link:** https://arxiv.org/abs/2302.07586
 - **Pdf link:** https://arxiv.org/pdf/2302.07586
 - **Abstract**
 There is a rapid increase in the number of mobile banking applications' users due to an increase in smart mobile devices. Mobile banking is a financial transaction and service offered through mobile devices. Almost all financial institutions now provide mobile banking services to their customers. However, the security of mobile banking applications is of huge concern because of the amount of personal data and information they collect. If an attacker gets hold of personal information, they can access bank payment or card accounts. This research aims to analyze the vulnerability of the UK digital banks' applications to identify vulnerabilities in the apps and proffer countermeasures that can help improve the security of the bank applications. Androbugs, a vulnerability scanner, was used to analyze the vulnerability of six digital banks' android applications. Starling, Monese, Atom bank, Transferwise, Monzo, and Revolut were scanned. All the scanned digital banks' applications have vulnerabilities; however, some have more vulnerabilities than others. For example, Revolut's mobile application has the highest number of identified vulnerabilities. Therefore, there is need for more security in the digital banks' applications as well as other mobile banking applications.
## Keyword: video retrieval
There is no result 
## Keyword: transfer functionmobile
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: volume data
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl2
There is no result 
### MSDA: Monocular Self-supervised Domain Adaptation for 6D Object Pose  Estimation
 - **Authors:** Dingding Cai, Janne Heikkilä, Esa Rahtu
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2302.07300
 - **Pdf link:** https://arxiv.org/pdf/2302.07300
 - **Abstract**
 Acquiring labeled 6D poses from real images is an expensive and time-consuming task. Though massive amounts of synthetic RGB images are easy to obtain, the models trained on them suffer from noticeable performance degradation due to the synthetic-to-real domain gap. To mitigate this degradation, we propose a practical self-supervised domain adaptation approach that takes advantage of real RGB(-D) data without needing real pose labels. We first pre-train the model with synthetic RGB images and then utilize real RGB(-D) images to fine-tune the pre-trained model. The fine-tuning process is self-supervised by the RGB-based pose-aware consistency and the depth-guided object distance pseudo-label, which does not require the time-consuming online differentiable rendering. We build our domain adaptation method based on the recent pose estimator SC6D and evaluate it on the YCB-Video dataset. We experimentally demonstrate that our method achieves comparable performance against its fully-supervised counterpart while outperforming existing state-of-the-art approaches.
### Tag-based annotation creates better avatars
 - **Authors:** Minghao Liu, Zeyu Cheng, Shen Sang, Jing Liu, James Davis
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV); Graphics (cs.GR)
 - **Arxiv link:** https://arxiv.org/abs/2302.07354
 - **Pdf link:** https://arxiv.org/pdf/2302.07354
 - **Abstract**
 Avatar creation from human images allows users to customize their digital figures in different styles. Existing rendering systems like Bitmoji, MetaHuman, and Google Cartoonset provide expressive rendering systems that serve as excellent design tools for users. However, twenty-plus parameters, some including hundreds of options, must be tuned to achieve ideal results. Thus it is challenging for users to create the perfect avatar. A machine learning model could be trained to predict avatars from images, however the annotators who label pairwise training data have the same difficulty as users, causing high label noise. In addition, each new rendering system or version update requires thousands of new training pairs. In this paper, we propose a Tag-based annotation method for avatar creation. Compared to direct annotation of labels, the proposed method: produces higher annotator agreements, causes machine learning to generates more consistent predictions, and only requires a marginal cost to add new rendering systems.
### Do Geometric Models Affect Judgments of Human Motion?
 - **Authors:** Jessica K. Hodgins, James F. O'Brien, Jack Tumblin
 - **Subjects:** Graphics (cs.GR)
 - **Arxiv link:** https://arxiv.org/abs/2302.07489
 - **Pdf link:** https://arxiv.org/pdf/2302.07489
 - **Abstract**
 Human figures have been animated using a wide variety of geometric models including stick figures, polygonal models, and NURBS-based models with muscles, flexible skin, or clothing. This paper reports on experiments designed to ascertain whether a viewer's perception of motion characteristics is affected by the geometric model used for rendering. Subjects were shown a series of paired motion sequences and asked if the two motions in each pair were "the same" or "different." The two motion sequences in each pair used the same geometric model. For each trial, the pairs of motion sequences were grouped into two sets where one set was rendered with a stick figure model and the other set was rendered with a polygonal model. Sensitivity measures for each trial indicate that for these sequences subjects were better able to discriminate motion variations with the polygonal model than with the stick figure model.
### LiveHand: Real-time and Photorealistic Neural Hand Rendering
 - **Authors:** Akshay Mundra, Mallikarjun B R, Jiayi Wang, Marc Habermann, Christian Theobalt, Mohamed Elgharib
 - **Subjects:** Graphics (cs.GR)
 - **Arxiv link:** https://arxiv.org/abs/2302.07672
 - **Pdf link:** https://arxiv.org/pdf/2302.07672
 - **Abstract**
 The human hand is the main medium through which we interact with our surroundings. Hence, its digitization is of uttermost importance, with direct applications in VR/AR, gaming, and media production amongst other areas. While there are several works for modeling the geometry and articulations of hands, little attention has been dedicated to capturing photo-realistic appearance. In addition, for applications in extended reality and gaming, real-time rendering is critical. In this work, we present the first neural-implicit approach to photo-realistically render hands in real-time. This is a challenging problem as hands are textured and undergo strong articulations with various pose-dependent effects. However, we show that this can be achieved through our carefully designed method. This includes training on a low-resolution rendering of a neural radiance field, together with a 3D-consistent super-resolution module and mesh-guided space canonicalization and sampling. In addition, we show the novel application of a perceptual loss on the image space is critical for achieving photorealism. We show rendering results for several identities, and demonstrate that our method captures pose- and view-dependent appearance effects. We also show a live demo of our method where we photo-realistically render the human hand in real-time for the first time in literature. We ablate all our design choices and show that our design optimizes for both photorealism and rendering speed. Our code will be released to encourage further research in this area.
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result
